### PR TITLE
Changed broken header background image to suitable replacement from unsplash.com (foggy lake scene)

### PR DESCRIPTION
**Pull request by FriscoTony**

### CODE REVIEW

File: 24 - Sticky Nav/style-START.css [hunk 1]
```diff
@@ -28,7 +28,7 @@ body {
 header {
   text-align: center;
   height: 50vh;
-  background: url(http://wes.io/iEgP/wow-so-deep.jpg) bottom center no-repeat;
+  background: url(https://source.unsplash.com/GKN6rpDr0EI/960x640) bottom center no-repeat;
```
Comment (harshitagrawal7733) [inline 24 - Sticky Nav/style-START.css:31]: background: url(https://f.v1.n0.cdn.getcloudapp.com/items/331o2t1L3t1y160m3N2c/Image%202016-11-24%20at%202.45.35%20PM.jpg) bottom center no-repeat;

Comment (harshitagrawal7733) [reply 24 - Sticky Nav/style-START.css:31]: Please Check that!

**Comment by wesbos** [comment]
thanks!

**Comment by harshitagrawal7733** [comment]
is that okay or working? @wesbos

**Pull request merged**